examples/petstore/config.yaml

```diff
@@ -29,9 +29,6 @@ security:
         audience: null
         claim_identity: sub
         claim_issuer: iss
-        claim_key_id: kid
-        header_name: Authorization
-        token_prefix: Bearer
```

Comment:
When I was trying out locally, these lines were giving errors and after removing them I was able to start the server. One of the possible reasons could be because of the old foca version in my local environment. I will try it out again and check if it works without removing them.

Reply:
I have tried rerunning by installing the foca again from the source code but still, it's showing the same error and works fine on removing these variables. I am attaching the screenshot below for your reference.
![Screenshot from 2021-02-05 08-34-56](https://user-images.githubusercontent.com/18120004/106984452-a4ed5900-678d-11eb-92dc-3ee282f386ea.png)

Reply:
I think I have refactored this recently and some of these may not be used anymore. I will double-check and address this when I review the PR later

Reply:
Ok, thanks a lot.